sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/datalake_file_client.hpp

```diff
@@ -233,9 +233,10 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
     explicit FileClient(
```

Comment:
Just for the record, `FileClient` contains a `BlockBlobClient` and a `BlobClient`, I don't think this is right. But I don't care as long as it works.